packages/core/admin/admin/src/components/PrivateRoute/tests/index.test.js

```diff
@@ -36,7 +36,7 @@ describe('PrivateRoute', () => {
     auth.setToken('access-token');
     renderApp();
     // Visit a protected route
-    history.push('/protected');
+    act(() => history.push('/protected'));
```

Comment:
In the instance you've highlighted we render the app first so we're "acting" on the react app. to be honest i wouldn't use history and instead use `MemoryRouter` following this guide from the react-router docs as opposed to `@testing-library/react` docs –  https://v5.reactrouter.com/web/guides/testing

It's something we can probably do alongside improving the FE test-utils like a global harness

Reply:
Interesting, but so I guess making such a change would be a different PR?

Reply:
Yeah definitely, because the `admin-test-utils` are in TS as well, we currently don't have defined FE TS tooling so I'm blocked on doing anything meaningful about it without exporting something from the `admin` package – either way, separate PR – but keen eye 👁️